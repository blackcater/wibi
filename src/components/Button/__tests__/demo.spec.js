import demoTest from '../../../../test/shared/demoTest';

demoTest('Button');
