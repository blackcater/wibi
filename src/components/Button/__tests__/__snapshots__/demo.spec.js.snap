// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button demo renders appearance demo correctly 1`] = `<div role="toolbar" class="vs-btn-toolbar"><button type="button" class="vs-btn vs-btn-default">Default<span class="vs-ripple-pond"><span class="vs-ripple" style="display: none;" enteractiveclass="vs-ripple-rippling"></span></span></button> <button type="button" class="vs-btn vs-btn-primary">Primary<span class="vs-ripple-pond"><span class="vs-ripple" style="display: none;" enteractiveclass="vs-ripple-rippling"></span></span></button> <button type="button" class="vs-btn vs-btn-link">Link</button> <button type="button" class="vs-btn vs-btn-subtle">Subtle<span class="vs-ripple-pond"><span class="vs-ripple" style="display: none;" enteractiveclass="vs-ripple-rippling"></span></span></button> <button type="button" class="vs-btn vs-btn-ghost">Ghost</button></div>`;

exports[`Button demo renders basic demo correctly 1`] = `<button type="button" class="vs-btn vs-btn-default">Default<span class="vs-ripple-pond"><span class="vs-ripple" style="display: none;" enteractiveclass="vs-ripple-rippling"></span></span></button>`;
